### PR TITLE
Fixing all gometalinter issues in bitvector. Increased code coverage …

### PR DESCRIPTION
**Pull request by tariq1890**
…TO 100%.

### CODE REVIEW

**Comment by pattyshack** [comment]
could you remove .gitignore?

File: container/bitvector/bitvector_test.go [hunk 1]
```diff
@@ -19,6 +20,7 @@ func (s *BitVectorSuite) SetUpTest(c *C) {
 	data := []byte{0xF0, 0x0F}
 	length := 12
```
Comment (pattyshack) [inline container/bitvector/bitvector_test.go]: just inline this

File: container/bitvector/bitvector_test.go [hunk 2]
```diff
@@ -19,6 +20,7 @@ func (s *BitVectorSuite) SetUpTest(c *C) {
...
 	s.vector = NewBitVector(data, length)
+	s.lVector = NewBitVector(data, 16)
```
Comment (pattyshack) [inline container/bitvector/bitvector_test.go]: vector & lVector shouldn't share the same byte array.  make a copy of the data

Comment (tariq1890) [reply container/bitvector/bitvector_test.go]: Sure!

File: container/bitvector/bitvector_test.go (hunk 1 above)
Comment (tariq1890) [reply container/bitvector/bitvector_test.go]: I am not sure what you mean by this

Comment (pattyshack) [reply container/bitvector/bitvector_test.go]: since length is only used immediately below, just do `s.vector = NewBitVector(data, 12)`

Comment (tariq1890) [reply container/bitvector/bitvector_test.go]: Sure. In that case, shall I Inline the data arrays as well?

Comment (tariq1890) [reply container/bitvector/bitvector_test.go]: s.vector = NewBitVector([]byte{0xF0, 0x0F}, 12)

	s.lVector = NewBitVector([]byte{0xF0, 0x0F}, 16)

Comment (pattyshack) [reply container/bitvector/bitvector_test.go]: sure

Comment (tariq1890) [reply container/bitvector/bitvector_test.go]: Cool :)

**Comment by tariq1890** [comment]
@pattyshack I've pushed the changes now.

**Pull request merged**